src/app/components/spinner/spinner.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from "@angular/core/testing";
+
+import { SpinnerService } from "./spinner.service";
+
+describe("SpinnerService", () => {
+    let service: SpinnerService;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({});
+        service = TestBed.inject(SpinnerService);
+    });
+
+    it("should be created", async () => {
+        await expect(service).toBeTruthy();
```

Comment:
I wonder why tests like this are necessary. I'm sure Angular `TestBed` works fine without our tests.